third_party/tsl/tsl/protobuf/dnn.proto

```diff
@@ -14,11 +14,12 @@ enum DataType {
   kHalf = 2;
   kInt8 = 3;
   kInt32 = 4;
-  kComplexFloat = 5;
-  kComplexDouble = 6;
-  kBF16 = 7;
-  kF8E5M2 = 8;
-  kF8E4M3FN = 9;
+  kInt64 = 5;
```

Comment:
Ideally, could we avoid changing values of previous versions?

Reply:
yea updated